core/src/test/java/apoc/schema/SchemasTest.java

```diff
@@ -49,8 +48,7 @@ public class SchemasTest {
 
     @Rule
     public DbmsRule db = new ImpermanentDbmsRule()
-            .withSetting(GraphDatabaseSettings.procedure_unrestricted, Collections.singletonList("apoc.*"))
-            .withSetting(GraphDatabaseInternalSettings.rel_unique_constraints, true);
```

Comment:
This is used in another class MetaTest.java. Should it be removed there, too?

Reply:
:O How did my search not find that 🙈 Thank you!